frontends/mit-open/src/components/RootTopicIcon/RootTopicIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import {
```

Comment:
I've added a PlatformIcons component in my local branch. I may combine these into an Icons component, if no objections.

Reply:
Ignore. They're logos not icons, so should be separate (though perhaps for all logos and also a component for all icons?).